app/mailers/application_mailer.rb

```diff
@@ -22,6 +22,28 @@ class ApplicationMailer < ActionMailer::Base
   helper :application # gives access to all helpers defined within `application_helper`.
 
 
+  # if there is a problem communicating with the MailGun REST server, log the problem
+  def self.deliver_mail(mail)
+
+    begin
+      super
+
+    rescue => e
```

Comment:
Probably should specify which errors we want to rescue here.  This will rescue StandardError and all its subclasses, but this seems a bit broad and might hide some errors that we would like to know about.